klippa/src/parsing_util.rs

```diff
@@ -86,6 +87,48 @@ pub fn parse_drop_tables(input_str: &str) -> Result<IntSet<Tag>, SubsetError> {
     Ok(result)
 }
 
+//parse input name_IDs string, which is a comma/whitespace-separated list of nameIDs that will be retained
+pub fn parse_name_ids(input_str: &str) -> Result<IntSet<NameId>, SubsetError> {
+    let mut result = IntSet::empty();
+
+    if input_str == "*" {
```

Comment:
Do you want to trim whitespace? - right now it looks like " * " won't behave as expected

Reply:
Thanks, appreciated! All these review comments are very helpful, I'll apply them to other parse_ functions as well